src/main/resources/application.yml

```diff
@@ -22,7 +22,9 @@ spring:
   profiles:
     group:
       test:
-        - "stdout"
+        - "dev"
+        - "auth-seed"
```

Comment:
Because we're connecting directly to HMPPS-Auth when we are running the application, only the test profile needs seed data for repository tests